graphics/spirograph_0.3_0.75_10.csv

```diff
@@ -0,0 +1,500 @@
+0.475,0.475
```

Comment:
please remove this file.

Reply:
Is there any problem

Reply:
please suggest me that where I have made mistake

Reply:
that file is not related to this PR.